_docs/sysadmin/update.md

```diff
@@ -12,6 +12,21 @@ page.**
 
 Please also see [Installation Version Notes](version_notes)
 
+### Announce Upcoming System Outage
+
+1. In advance of system updates or system maintenance, you may wish to
+   announce the outage to your users.  To do this, edit the
+   `/usr/local/submitty/config/submitty.json` file adding/editing the
+   `"system_message"` field:
+
+    ```
+    "system_message" : "We will have brief maintenance outage on Tuesday at 8am."
+    ```
+
+    ![](/images/system_outage_message.png)
```

Comment:
we are capturing the info of editing this file [here](https://submitty.org/sysadmin/system_customization#show-system-message-to-all-users) as well. Do we want to keep the duplication or have one link to the other? I do think having it shown here is definitely good.

Reply:
I attempted to search for it and didn't find it before adding the new documentation. Whoops.